api/api.proto

```diff
@@ -3258,6 +3272,18 @@ message SdkGoogleCredentialResponse {
   string project_id = 2;
 }
 
+// Defines the response for NFS credential
+message SdkNfsCredentialResponse {
```

Comment:
Why do we need to return all the input parameters back?

Reply:
We dont have to. Its sort of data that helps confirmation/enumerate for callers